### ISSUE TITLE
Unknown dictionary for pre-trained models

### ISSUE THREAD

**Issue by nathanb97** [cristinalunaj/MMEmotionRecognition#6]
Hello,
For pre-trained models we don't have the dictionary to which predicted value translators ({0: "Neutral", 1: "fear" ...}).

Your code rebuilds the dictionary based on the Ravdess data-set.
But it would be much useful if we had this dictionary. (Ravdess is long to download, and long to pre-process, and the fear of not having the same dictionary).

Thank you very much for your time.
Preferably for best model RAVDESS_AUs-fold_1_outof_5-0.6662-2021_10_21_17_55.model

**Comment by nathanb97** [comment]
Ok sorry after reading of your codes you 
It seems that you classify the labels in alphabetical order.
And so the dictionary would be :
```
{'angry': 0, 'calm': 1, 'disguted': 2, 'fear': 3, 'happy': 4, 'neutral': 5, 'sad': 6, 'surprised': 7}
```
I am waiting for your confirmation before closing :+1:

**Comment by cristinalunaj** [comment]
Hello! Sorry I have been a little busy these days _:) 
Yes! Exactly, outputs of trained models follow the dict that you say: 
{'Angry': 0, 'Calm': 1, 'Disgust': 2, 'Fear': 3, 'Happy': 4, 'Neutral': 5, 'Sad': 6, 'Surprise': 7}
It is necessary to make a processing step to match them with the ground-truths of the RAVDESS dataset, that I do in Wav2VecEval.py
Thanks for  your comment, I will include in the README.
